src/prisma/schema.prisma

```diff
@@ -402,6 +415,32 @@ model StripeSubscription {
   @@index([spaceId])
 }
 
+model SpaceSubscriptionPayment {
+  id              String   @id @default(uuid()) @db.Uuid
+  paidTokenAmount String
+  spaceId         String?  @db.Uuid
+  space           Space?   @relation(fields: [spaceId], references: [id], onDelete: SetNull)
+  createdAt       DateTime @default(now())
+}
+
+model SpaceSubscriptionContribution {
+  id              String         @id @default(uuid()) @db.Uuid
+  spaceId         String?        @db.Uuid
+  space           Space?         @relation(fields: [spaceId], references: [id], onDelete: SetNull)
+  userId          String?        @db.Uuid
+  user            User?          @relation(fields: [userId], references: [id], onDelete: SetNull)
+  walletAddress   String
+  paidTokenAmount String
```

Comment:
Suggest 'devTokenAmount' just in case u want to also store like the sourceTokenAmount or something in the future